### PR TITLE
Add Bonobo tool and other stuff

### PR DESCRIPTION
**Pull request by matty45**

### CODE REVIEW

**Comment by matty45** [comment]
that last commit should have been more informative, i moved the modding limitations to a new folder so that people know its aimed towards MCC

**Comment by matty45** [comment]
Managed to get what i talked about in the previous comment working properly! :D

**Comment by MegaMech** [comment]
Having a sub-folder adds unnecessary clicks. Perhaps in the future this would be warranted. As for now it would be better to just put:
```
# PC

# Xbox
```

**Comment by matty45** [comment]
halo cea now appears correctly

File: docs/Halo/CEA/HaloCEA.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Halo: CE Anniversary
+title: "Halo: CE Anniversary"
```
Comment (MegaMech) [inline docs/Halo/CEA/HaloCEA.md]: I can confirm this solution works.

Comment (matty45) [reply docs/Halo/CEA/HaloCEA.md]: also added this information to contributing

File: docs/Contributing/Contributing.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ has_toc: false -- Disables table of contents auto-generation.
 grand_parent: Tools -- The title of the parent of the parent
  ---
  ```
+* Use quotes if your field has special characters such as `:`.
```
Comment (MegaMech) [inline docs/Contributing/Contributing.md]: Insert quotes if special characters such as `:` are required.

Comment (matty45) [reply docs/Contributing/Contributing.md]: that makes more sense

File: docs/Tools/Bonobo/Bonobo.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+layout: default
+title: Bonobo
+parent: Tools
+has_children: true
+has_toc: false
+nav_order: 7
+---
+# Bonobo
+
+Bonobo extracts animations from halo games and decompiles them.
```
Comment (MegaMech) [inline docs/Tools/Bonobo/Bonobo.md]: Bonobo extracts and decompiles Halo animations.
Is it possible to add a couple more sentences?

File: docs/Tools/Bonobo/Bonobo.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Accessing extracted animations requires an edition of 3DS Max from 2013 to current.
+
+#### List of supported Halo games
```
Comment (MegaMech) [inline docs/Tools/Bonobo/Bonobo.md]: `#### Compatible with:`

Comment (matty45) [reply docs/Tools/Bonobo/Bonobo.md]: Should i add that to adjutant too?

Comment (MegaMech) [reply docs/Tools/Bonobo/Bonobo.md]: I already did

Comment (matty45) [reply docs/Tools/Bonobo/Bonobo.md]: oh, ok

File: docs/Tools/Bonobo/Bonobo.md (hunk 3 above)
Comment (matty45) [reply docs/Tools/Bonobo/Bonobo.md]: yea

**Comment by no1dead** [comment]
Could this PR be resubmitted? As it isn't gonna cleanly merge into the current branch, and will muddy up the actual commit history as there's a lot of reverting and re adding things, and single line commits that will bloat the commit history.

**Pull request closed without merging**